src/static/app/common/factory/ApiFactory.js

```diff
@@ -87,6 +87,9 @@ export default class ApiFactory {
         body: schemaPolicyData
       });
   }
+  getSubject(projectName) {
```

Comment:
Please rename function name `getSchemapolicyStatementSubject` to prevent function name

Reply:
@EgumaYuto 
fixed.
eea814214e67a0c30094390bcc8e5943c0e5d68f